test.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -ex
```

Comment:
I think I prefer having `-x` set. Most of the time, when I'm looking at the logs, it's because something's gone wrong. Without `-x`, it can be hard to figure out exactly where things broke. Also, with `-x`, I can cut and paste commands to rerun them when I need to.